src/api/lib/backend/api/published.rb

```diff
@@ -20,6 +20,13 @@ def self.build_id(project_name, repository_name)
           Xmlhash.parse(response)['buildid']
         end
       end
+
+      def self.published_repository_exist?(project, repository)
```

Comment:
To be consistent, I would pass as parameters only the name of the project and the name of the repository. Then the next [2 lines](https://github.com/openSUSE/open-build-service/pull/9427/files#diff-51b1b7f7dd955063df614ebf0b46831eR25-R26) wouldn't be needed.

Reply:
Done